includes/rest-api/routes/content-model-field.php

```diff
@@ -104,43 +105,96 @@ function dispatch_update_content_model_field( WP_REST_Request $request ) {
 	if ( isset( $params['type'] ) && $params['type'] === 'multipleChoice' && ! $params['choices'] ) {
 		return new WP_Error(
 			'wpe_invalid_multi_options',
-			'Multiple Choice update failed. Options need to be created before updating a Multiple Choice field.',
+			__( 'Multiple Choice update failed. Options need to be created before updating a Multiple Choice field.', 'atlas-content-modeler' ),
```

Comment:
```suggestion
			__( 'Multiple Choice update failed. Choices need to be created before updating a Multiple Choice field.', 'atlas-content-modeler' ),
```

Should we say "Choices" instead of "Options" here, since the UI calls them choices?